code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Dispenser.dm

```diff
@@ -214,25 +269,28 @@
 /datum/reagent/oxygen/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
 	if(alien == IS_VOX)
 		M.adjustToxLoss(removed * 3)
-
+*/
```

Comment:
Acetone, hydrazine, ammonia, and hydrochloric acid are all actually real, stable compounds, and make more sense than attempting to mix a liquid form of a gas into medicine.